docs/features/opencdc-record.mdx

```diff
@@ -0,0 +1,132 @@
+* `.Key` represents a value that should identify the entity (e.g. database row).
+* `.Metadata` contains additional information regarding the record.
+* `.Payload.Before` holds the payload before the operation ocurred. These could be present in operations such as `update` and `delete`.
+* `.Payload.After` holds the payload after the operation ocurred. These could be present in operations such as `create`, `snapshot` or `update`.
+
+## Representation
+
+Conduit relies on [protobuf](https://protobuf.dev/) when it comes to defining an OpenCDC record to benefit from the several advantages that it provides. Its definition can be found in the [Buf Schema Registry](https://buf.build/conduitio/conduit-commons/docs/main:opencdc.v1).
+
+When processing records in Conduit, you can always expect this format:
+
+```json
+{
+    "position": "c3RhbmRpbmc=",
+    "operation": "update",
+    "metadata": "{
+        "file.path": "./example.in",
+        "opencdc.readAt": "1663858188836816000",
+        "opencdc.version": "v1"
+    }",
+    "key": "cGFkbG9jay1rZXk=",
+    "payload": {
+        "before": "eWVsbG93",
+        "after": {
+            "bool": true,
+            "float32": 1.2,
+            "float64": 1.2,
+            "int": 1,
+            "int32": 1,
+            "int64": 1,
+            "string": "orange"
+        }
+    }
+}
+```
+
+:::note
+`.Position`, `.Key`, and `.Payload.Before` are represented as `Base64` encoded in the example above because these will be a byte slice when represented as JSON.
+:::
+
+## Metadata constants
+
+As part of an OpenCDC record, there will be a set of constants provided for some metadata fields that will vary depending on the connector. These constants can be common to all **OpenCDC** records as part of our standard, some related to our own **Conduit** plugins, and others that will be provided by the **Connector SDK** independently. These constants can be useful to define conventions that will be then used by Conduit to expand its functionality. Here are the ones you can find:
+
+### OpenCDC
+
+- `opencdc.createdAt` can contain the time when the record was created in the 3rd party system. The expected format is a Unix timestamp in nanoseconds.
+- `opencdc.readAt` can contain the time when the record was read from the 3rd party system. The expected format is a Unix timestamp in nanoseconds.
+- `opencdc.version` contains the version of the OpenCDC format (e.g., "v1"). This field exists to ensure the OpenCDC format version can be easily identified in case the record gets marshaled into a different untyped format (e.g. JSON).
+
+```json
+{
+    ...
+    "metadata": "{
+        "opencdc.createdAt": "1663858188836816000",
+        "opencdc.readAt": "1663858188836816000",
+        "opencdc.version": "v1",
+        ...
+    }",
+    ...
+}
+```
+
+### Conduit
```

Comment:
Here we're missing [3 metadata fields](https://github.com/ConduitIO/conduit-commons/blob/a6adcbfe1168a50f5f42fdeafa0353eae789dd8e/opencdc/metadata.go#L59-L67), which are populated when a record is sent to the DLQ. Even though they are not in the proto files (yet), they are important, as they give you insight into why a record landed in the DLQ.